earthly/postgresql/Earthfile

```diff
@@ -43,10 +43,6 @@ postgres-base:
     # Get refinery
     COPY rust-tools+tool-refinery/refinery /bin
 
-    # Get dbviz
```

Comment:
Instead of trying to run another container, could you just copy the necessary tools for schemaspy into postgres-base here?

Reply:
packages not added in postgres-base